Sources/Attribution/AttributionFetcher.swift

```diff
@@ -23,15 +23,6 @@ import WatchKit
 import AdServices
 #endif
 
-enum AttributionFetcherError: Error {
-
-    case identifierForAdvertiserUnavailableForPlatform
-    case identifierForAdvertiserFrameworksUnavailable
-    case adServicesNotAvailable
-    case adServicesTokenFetchError
```

Comment:
I removed these 2 cause they aren't used, but it was harder to see cause this was an `internal` type